config/config.properties

```diff
@@ -91,6 +91,21 @@ pravegaservice.dataLogImplementation=BOOKKEEPER
 # Default value: HDFS
 # pravegaservice.storageImplementation=HDFS
 
+# Storage NO-OP Mode: in No-Op mode, user stream segment writing is no-oped; user stream segment reading is not supported.
+# This mode is used to avoid storage interference in testing while still keep the system functioning as usual.
```

Comment:
Suggestion for rephrase: `This mode is useful to test the performance of Pravega against Tier 1 (e.g., Bookkeeper) without interferences related to moving data to Tier 2.`